base/task/sequence_manager/task_queue_impl.cc

```diff
@@ -1346,7 +1346,7 @@ void TaskQueueImpl::SetThrottler(TaskQueue::Throttler* throttler) {
 
 void TaskQueueImpl::ResetThrottler() {
   main_thread_only().throttler = nullptr;
-  LazyNow lazy_now(sequence_manager_->main_thread_clock());
+  LazyNow lazy_now(sequence_manager_->any_thread_clock_maybe_events_disallowed());
```

Comment:
Why did we change it from `main` to `any`? 
(I assume, they are quite interchangeable?)

Reply:
This is for simplicity, the any_thread version subsumes the main_thread version as they're both atomic accesses with different memory order constraints.